<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sel_dev</class>
 <widget class="QDialog" name="sel_dev">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>136</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>451</width>
     <height>108</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="OK">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>65</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Close">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>40</y>
     <width>65</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Radio_port0_SSD">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>48</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Port 0 SSD                              dev/nvme0n1  </string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Radio_port1_SSD">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>78</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>port 1 SSD                              dev/nvme1n1  </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
